openmicroscopy/openmicroscopy
# Make Java 1.7 the minimum supported version

Note this does not change jarsigner, which can be upgraded to use 1.7 features separately.

- Bump minimum version numbers from 1.6 to 1.7
- Drop special cased building of Java6 MacOS X app bundles

--------

Testing:

- Check travis and CI builds are green.  This shouldn't cause any breakage, but it's possible if we have any CI nodes using Java6 they will fail the build and need upgrading
- Check client and server are still fully functional; we're bumping the JVM minimum version, so there might be potential for issues with the 1.7 bytecode (unlikely, but still needs testing)
- Check all clients, particularly Mac client in case I broke the packaging.
